Mendelugram/MendelugramTests/ViewModelTests/PhotoViewModelTests.swift

```diff
@@ -0,0 +1,75 @@
+//
+//  PhotoViewModelTests.swift
+//  MendelugramTests
+//
+//  Created by David Aldorf on 28/04/2019.
+//
+
+import XCTest
+@testable import Mendelugram
+
+//protocol PhotoViewModeling {
```

Comment:
What is a purpose of this commented-out implementation?

Reply:
It's just to show them which protocol should be tested.

Reply:
Ok. I have no problem with that.  I was just curious if it is your standard convention or if you decided for this approach just in this case.